src/lib.rs

```diff
@@ -40,10 +40,10 @@ extern crate graphene;
 extern crate gsk4 as gsk;
 extern crate pango;
 
-#[cfg(feature = "futures")]
+#[cfg(any(feature = "futures", feature = "dox"))]
```

Comment:
```suggestion
#[cfg(feature = "futures")]
```

"futures" is implied by "dox" in Cargo.toml. Just trying to be consistent with gtk3 here.